packages/expo-screen-capture/src/ExpoScreenCapture.web.ts

```diff
@@ -2,6 +2,9 @@ export default {
   get name(): string {
     return 'ExpoScreenCapture';
   },
+  async isAvailableAsync(): Promise<false> {
+    return false;
+  },
```

Comment:
I don't think this is hit, since `ScreenCapture.ts` doesn't call into ExpoScreenCapture.isAvailableAsync(). But since `preventScreenCapture` and `allowScreenCapture` do nothing on web, we can just remove them from `web.ts` and instead check & maybe throw in each method like this:
```
if (!ExpoScreenCapture.preventScreenCaptureAsync) {
    throw new UnavailabilityError('expo-screen-capture', 'preventScreenCaptureAsync');
  }
```
then we can just remove this function in `web.ts` as well, and `ScreenCapture.isAvailableAsync` should work as expected (returning false)

Reply:
Done!